### DISCUSSION TITLE
Help with PWMERGE

### DISCUSSION THREAD

**Discussion by maccadia** [ltsp/ltsp#756] [Q&A] [answered]
Hi,

I would like to remove all local users present on the server from the clients.

Here is my `/etc/ltsp/ltsp.conf` : 
```
[common]
PWMERGE_SUR=""
PWMERGE_SGR=""
```

After running `ltsp initrd`, I still have the server's user on the clients. I'm confused.

Version :
ltsp 22.01-1~ubuntu20.04.1

**Comment by alkisg** [comment, marked as answer]
In https://github.com/ltsp/ltsp/blob/main/ltsp/client/init/54-pam.sh#L13 try to replace:

```diff
-        ${PWMERGE_SUR:+"--sur=$(re eval_percent "$PWMERGE_SUR")"} \
-        ${PWMERGE_SGR:+"--sgr=$(re eval_percent "$PWMERGE_SGR")"} \
-        ${PWMERGE_DUR:+"--dur=$(re eval_percent "$PWMERGE_DUR")"} \
-        ${PWMERGE_DGR:+"--dgr=$(re eval_percent "$PWMERGE_DGR")"} \
+        ${PWMERGE_SUR+"--sur=$(re eval_percent "$PWMERGE_SUR")"} \
+        ${PWMERGE_SGR+"--sgr=$(re eval_percent "$PWMERGE_SGR")"} \
+        ${PWMERGE_DUR+"--dur=$(re eval_percent "$PWMERGE_DUR")"} \
+        ${PWMERGE_DGR+"--dgr=$(re eval_percent "$PWMERGE_DGR")"} \
```

...because as it is now, the shell part of `ltsp init` doesn't forward empty variables to `pwmerge`.

If that doesn't work, just put a non-existing regex, e.g. `PWMERGE_SUR="^no user$"`.

**Reply by maccadia** [reply]
Thanks.

The 54-pam.sh modification doesn't work for me. Using a non-matching regex is fine with me.
